utils/sl/fake_scons/SCons/Script/__init__.py

```diff
@@ -293,6 +293,10 @@ def CheckFlag(self, *args, **kw):
         """Fake CheckFlag"""
         return True
 
+    def CheckASan(self, *args, **kw):
```

Comment:
(pylint-unused-argument) Unused argument 'args'
(pylint-unused-argument) Unused argument 'kw'
(pylint-no-self-use) Method could be a function